src/webview/create-component/pages/createComponent.tsx

```diff
@@ -21,34 +23,41 @@ interface VSCodeMessage {
     componentName?: string;
 }
 
-function SelectStrategy() {
+function SelectStrategy({ setCurrentView }) {
     const createComponentStyle = useCreateComponentStyles();
     return (
         <>
             <div className={createComponentStyle.headerContainer}>
-                <Typography variant="h5">Create Component</Typography>
+                <Typography variant='h5'>Create Component</Typography>
             </div>
             <div
                 style={{
                     display: 'flex',
                     flexDirection: 'row',
                     gap: '4em',
                     justifyContent: 'center',
+                    marginTop: '5em'
                 }}
             >
                 <OptionCard
-                    title="From Existing Local Codebase"
-                    description="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor."
+                    pageId='fromLocalCodeBase'
+                    setCurrentView={setCurrentView}
+                    title='From Existing Local Codebase'
+                    description='Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.'
```

Comment:
Please remove this text. I think it's okay not to have any description text, I don't think there is anything we need to explain.